src/App.js

```diff
@@ -1,13 +1,69 @@
-import React from 'react';
-import { BrowserRouter as Router, Routes, Route } from 'react-router-dom';
-import Greeting from './components/Greeting';
-
-const App = () => (
-  <Router>
-    <Routes>
-      <Route exact path="/" element={<Greeting />} />
-    </Routes>
-  </Router>
-);
+import React, { useEffect, useState } from 'react';
+import { BrowserRouter as Router, Route, Routes } from 'react-router-dom';
+import { useDispatch } from 'react-redux';
+import SplashPage from './components/SplashPage';
+import HomePage from './components/HomePage';
+import LoginForm from './components/sessions/LoginForm';
+import RegistrationForm from './components/sessions/RegistrationForm';
+import HouseForm from './components/addHouse';
+import ReservationsList from './components/ReservationsList';
+import ReservationForm from './components/ReservationForm';
+import { fetchHouses } from './redux/houses/housesSlice';
+import NavBar from './components/NavBar';
+import HouseDetails from './components/HouseDetails';
+import DeleteHouse from './components/DeleteHouse';
+
+const App = () => {
```

Comment:
- [ ] In the mobile view, there is no navigation bar:

![image](https://github.com/mercymugambi/frontend_house_reservations_react_app/assets/57421823/f781be96-f96d-4cc1-8cdb-234b80e0c0d2)

![image](https://github.com/mercymugambi/frontend_house_reservations_react_app/assets/57421823/973722d4-1dc3-4f7d-82a1-148bc48825a0)

![image](https://github.com/mercymugambi/frontend_house_reservations_react_app/assets/57421823/7a9b5e88-c39d-4a1a-8d4b-58234d9f514a)

Kindly, add some kind of navigation so your app can be used on mobile.

That way, your UX will improve for your users. 👍